.github/workflows/build.yml

```diff
@@ -10,9 +10,40 @@ jobs:
     runs-on: ${{ matrix.os }}
```

Comment:
A nitpick, but I prefer having this stuff more tight: `${{matrix.os}}`.
Would be nice to have prettier in this repo as well, to format YAML and other things.

Reply:
> A nitpick, but I prefer having this stuff more tight: `${{matrix.os}}`.

As GHA expressions can become quite complex as they support complex JavaScript, I find that the added whitespace increases legibility. But I don't care much one way or the other, so I removed the extraneous whitespace in 70955f95fb2e0575d693d7845c5de614caf725cf.

> Would be nice to have prettier in this repo as well, to format YAML and other things.

Agreed. Do you have a specific configuration you'd like Prettier to use, then a PR with that configuration would be great!